### PR TITLE
[0.8.0] Allow filtering jobs by Last Run Status

### PR DESCRIPTION
**Pull request by anukul**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Other (please describe):

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] If it's a backend change, tests for the changes have been added and `go test ./...` runs successfully from the root folder.
- [x] If it's a frontend change, Prettier has been run
- [ ] Docs have been reviewed and added / updated if needed

## What is the current behavior?

n/a

## What is the new behavior?

Users can filter jobs by the status of their last run

https://user-images.githubusercontent.com/44864521/130279761-eacb0c39-d276-4efe-8027-4fcaa53eb276.mov

## Technical Spec/Implementation Notes

very hacky

### CODE REVIEW

File: dashboard/src/main/home/cluster-dashboard/chart/ChartList.tsx [hunk 1]
```diff
@@ -210,7 +229,13 @@ const ChartList: React.FunctionComponent<Props> = ({
       });
     }
     return () => (isSubscribed = false);
-  }, [namespace, currentView]);
+  }, [
+    sortType,
+    lastRunStatus,
+    JSON.stringify(jobChartsLastRunStatus),
```
Comment (jnfrati) [inline dashboard/src/main/home/cluster-dashboard/chart/ChartList.tsx:235]: Quick question here, why using stringify instead of just the variable?

Comment (anukul) [reply dashboard/src/main/home/cluster-dashboard/chart/ChartList.tsx:235]: Passing objects in dependencies [doesn't work](https://twitter.com/dan_abramov/status/1104414272753487872) due to the referential equality check.

Comment (anukul) [reply dashboard/src/main/home/cluster-dashboard/chart/ChartList.tsx:235]: Although I did put this together very quickly, so I'm open to suggestions.

**Comment by anukul** [comment]
Sorry I think I need to take more time and redo this.

**Pull request closed without merging**